distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/mock/csw/FederatedCswMockServer.java

```diff
@@ -38,8 +38,7 @@
  * and federated query responses for a csw endpoint.
  */
 public class FederatedCswMockServer {
-    private static final XLogger LOGGER =
-            new XLogger(LoggerFactory.getLogger(FederatedCswMockServer.class));
+    protected static final Logger LOGGER = LoggerFactory.getLogger(FederatedCswMockServer.class);
```

Comment:
I would say yes. No need to go hunting them out though - I wouldn't add changes in your PR for files that you aren't actually working in just for the logger.